inkscape/svg2shenzhen/export.py

```diff
@@ -628,32 +628,31 @@ def exportEdgeCut(self, kicad_mod=False):
 
             layer_trans = layer.get('transform')
             if layer_trans:
-                layer_m = simpletransform.parseTransform(layer_trans)
+                layer_m = Transform(layer_trans).matrix
```

Comment:
This can be just `layer.transform.matrix`, it will return a clean transform even if it's empty.

Reply:
I'm on thin ice here. I'm not familiar with the Inkscape APIs and don't know how to test this.

Reply:
Docs wise transform is a shape level attribute object (like style) example of use https://gitlab.com/inkscape/extensions/-/blob/master/inkex/elements/_base.py#L430